src/views/main-view.js

```diff
@@ -132,7 +132,8 @@ MainView.prototype._initialize = function () {
   this._sendToDefaultView();
 };
 
-MainView.prototype._onChangeActivePaymentMethodView = function (id) {
+MainView.prototype._onChangeActivePaymentMethodView = function (data) {
```

Comment:
Shouldn’t this get changed to ‘onChangeActiveView’ too?

Reply:
Yep!